tests/Unit/Api/AbstractApiTest.php

```diff
@@ -129,7 +129,8 @@ public function getJsonDecodingFromGetMethodData()
             ['{"foo_bar": 12345}', null, ['foo_bar' => 12345]], // test decode by default
             ['{"foo_bar": 12345}', false, '{"foo_bar": 12345}'],
             ['{"foo_bar": 12345}', true, ['foo_bar' => 12345]],
-            ['', true, null], // test decode with empty body
+            'Empty body, JSON decode: false' => ['', false, false],
```

Comment:
Instead of having the comment in the key please move it at the end of the line like 
`['', false, false], // Empty body, JSON decode: false`

Reply:
This makes the test suite less explicit about the failed case, but no problem. I'll place the context information as comment.

Reply:
Wow, I didn't know that phpunit will use the key for the description in a failing test. :exploding_head: 
Please keep it as it is. :+1: